### PR TITLE
Adds .find! to RedPotion

### PR DESCRIPTION
**Pull request by squidpunch**

### CODE REVIEW

**Comment by markrickert** [comment]
Looks like the build is failing. I restarted it but got the same failure.

**Comment by squidpunch** [comment]
I think the failure is unrelated to the changes here, but I can check into it hopefully later tonight.

File: lib/project/ext/object.rb [hunk 1]
```diff
@@ -10,4 +10,9 @@ def device
   def find(*args) # Do not alias this, strange bugs happen where classes don't have methods
     rmq(*args)
   end
+
+  def find!(*args)
+    #TODO this can be rmq.find!(*args) when released in RMQ
```
Comment (twerth) [inline lib/project/ext/object.rb]: No, `rmq.find!` is different. `rmq(a).find!(b)` <- this find is for all children/grandchildren.
This find is: rmq(a).get. I'd change that to this too, btw, no need for one extra method call.

Thanks David.

Comment (squidpunch) [inline lib/project/ext/object.rb]: :+1: pushing an update, good call.

**Pull request merged**